toonz/sources/toonz/preferencespopup.cpp

```diff
@@ -1680,6 +1688,10 @@ QWidget* PreferencesPopup::createGeneralPage() {
                            &PreferencesPopup::onWatchFileSystemClicked);
 
   bool ret = true;
+
+  ret = ret && connect(pathAliasPriorityCB, SIGNAL(currentIndexChanged(int)),
+      this, SLOT(onPathAliasPriorityChanged()));
```

Comment:
OH, I seem to have forgotten to include this callback. Thank you!
Instead of calling onPathAliasPriorityChanged as a SLOT function, please do the following:

` m_onEditedFuncMap.insert(pathAliasPriority, &PreferencesPopup::onPathAliasPriorityChanged);`

This ensures that `onPathAliasPriorityChanged()` is called **after** the Preferences value is changed within `onChange()`.